app/views/components/_govspeak-editor.html.erb

```diff
@@ -1,5 +1,7 @@
 <%
+  binding.pry
```

Comment:
Looks like this was left behind by mistake.

Reply:
oops yep good spot 🤦🏻‍♂️